common/src/main/webapp/less/style.less

```diff
@@ -34,6 +34,10 @@ body {
   width: 100vw;
 }
 
+a {
+  color: lightblue;
```

Comment:
Do we want to change every link? At any rate this should be a `var`

Reply:
yes, agreed, I'm not familiar with how theming works and vars, let me check